mimecast/komand_mimecast/actions/create_managed_url/action.py

```diff
@@ -31,7 +29,7 @@ def run(self, params={}):
 
         # Mimecast request
         mimecast_request = util.MimecastRequests()
-        response = mimecast_request.mimecast_post(url=url, uri=CreateManagedUrl._URI,
+        response = mimecast_request.mimecast_post(url=url, uri=self.connection._create_managed_url_URI,
```

Comment:
Functions/properties prefixed with a `_` are the Python equivalent of `private` and shouldn't be accessed from outside their scope